agent/app/agent_capability_windows_test.go

```diff
@@ -221,6 +226,64 @@ func TestSupportedCapabilitiesWindows(t *testing.T) {
 	}
 }
 
+func TestFirelensConfigCapabilitiesWindows(t *testing.T) {
```

Comment:
same comment as above. Can the existing test https://github.com/aws/amazon-ecs-agent/blob/4d4c4afffc7283089c74d4513a9c5d90d3842c43/agent/app/agent_capability_unix_test.go#L812 be moved to agent_capability_test.go file instead?

Reply:
Actually it appears that this was not the reason for the failed test.